kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/BaseAuthzGroup.java

```diff
@@ -1056,6 +1068,14 @@ public void removeMember(String user)
 		if (m_lazy) baseAuthzGroupService.m_storage.completeGet(this);
 
 		m_userGrants.remove(user);
+		
+		//send message to (ignite) MicrosoftMessagingService
+		this.baseAuthzGroupService.microsoftMessagingService().send(MicrosoftMessage.REMOVE_MEMBER_FROM_AUTHZGROUP, MicrosoftMessage.builder()
```

Comment:
Shall we try/catch this or is it safe?

Reply:
It's safe. Try-catch is done inside MicrosoftMessagingService.send